Core/include/Acts/Fitter/GainMatrixSmoother.hpp

```diff
@@ -36,10 +37,22 @@ class GainMatrixSmoother {
           getDefaultLogger("GainMatrixSmoother", Logging::INFO).release()))
       : m_logger(std::move(logger)) {}
 
+  /// @Operater for Kalman smoothing
```

Comment:
```suggestion
  /// Operater for Kalman smoothing.
```
I think the `@` is a typo here?

Reply:
Ah, right. Removed.